Makefile

```diff
@@ -78,9 +78,11 @@ libflux: libflux/target/debug/libflux.a libflux/target/debug/liblibstd.a
 # command line interface than the gnu equivalent.
 libflux/target/debug/libflux.a:
 	cd libflux && $(CARGO) build -p flux $(CARGO_ARGS)
+	echo "const char uniq_libflux[] = \"`cat $@ | md5sum | awk '{print $$1;}'`\";" > libflux/go/libflux/libflux.c
 
 libflux/target/debug/liblibstd.a:
 	cd libflux && $(CARGO) build -p libstd $(CARGO_ARGS)
+	echo "const char uniq_liblibstd[] = \"`cat $@ | md5sum | awk '{print $$1;}'`\";" > libflux/go/libflux/liblibstd.c
```

Comment:
I had some trouble downloading `md5sum` so I used `shasum` here when I was borrowing your change. Otherwise, thanks for this. Improves workflow for sure!

Reply:
Great! Yeah `shasum` is probably the better choice for this hack. I'm noticing now that out of the two, only `shasum` comes out of the box on both linux and mac.